buildout.cfg

```diff
@@ -13,6 +13,10 @@ recipe = zc.recipe.egg
 eggs =
     Sphinx
     docutils
+    ZODB
+    j1m.sphinxautointerface
```

Comment:
Also, the default order (hash) is silly. In any case, I'm too lazy to change this now. :) I wish I'd found this earlier.